### ISSUE TITLE
HTML Entities in language link

### ISSUE THREAD

**Issue by chyne** [wet-boew/cdts-sgdc#353]
In my appTop section I was rendering the lngLinks text as below, with HTML Entity &amp;#231; for "ç".

This rendered as "Français" up to 4.0.44. In 4.0.44 this now renders as "Fran&amp;#231;ais".

Is this change intentional?

```
lngLinks: [
{
   lang: "fr",
   href: "/RoeWebMVC/pub/Language/Change",
   text: "Fran&#231;ais"
}]
```

**Comment by plup2** [comment]
Not entirely intentional, no.  In this release we switched templating library and the intention was to preserve the character escaping where it existed in the old sources.  However, we failed to notice that the new library escapes strings more aggressively and our tests didn't include HTML entities.

This will be corrected as soon as we can. If you're using CDTS by pointing to the GCintranet template environment (hosted on templates.service.gc.ca or ssl-templates.services.gc.ca), we're hoping to have that patched in the next couple of days.  If you're pointing to the GCweb environment (ie hosted on Akamai / canada.ca), hoping to have the patch in place by end of next week.
